src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -32,21 +40,43 @@ const DefiHeader = () => {
 
 const FoxFarmCTA = () => {
   const translate = useTranslate()
+  const history = useHistory()
+  const location = useLocation()
   const { farmingAprV4, isFarmingAprV4Loaded } = useFarmingApr()
   const { lpApr, isLpAprLoaded } = useLpApr()
+  const featureFlags = useAppSelector(selectFeatureFlags)
   const ethAsset = useAppSelector(state => selectAssetById(state, 'eip155:1/slip44:60'))
   const foxAsset = useAppSelector(state =>
     selectAssetById(state, 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'),
   )
   const { icon: ethAssetIcon } = ethAsset
   const { icon: foxAssetIcon } = foxAsset
   const hoverBg = useColorModeValue('gray.100', 'gray.750')
+
+  const handleClick = useCallback(() => {
+    if (!(featureFlags.FoxLP && featureFlags.FoxFarming)) {
+      window.location.replace('https://fox.shapeshift.com/fox-farming')
```

Comment:
This won't open in a new tab and is not the prettiest (unless we use `window.open()` which might be blocked by Adblock-like extensions) but it will be the easiest to refactor and is a cleaner implementation IMO vs. spreading conditional `href` and `isExternal` `<Link />` props.

Reply:
agreed, this is also *actually* temporary